mysys/CMakeLists.txt

```diff
@@ -89,6 +89,11 @@ ELSEIF(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|AARCH64")
   IF(CMAKE_COMPILER_IS_GNUCC)
     include(CheckCXXSourceCompiles)
 
+    # Check if NEON intrinsics are available and does not
+    # print error 'NEON intrinsics not available with the soft-float ABI.
+    # Please use -mfloat-abi=softfp or -mfloat-abi=hard'
+    CHECK_INCLUDE_FILES(arm_neon.h HAVE_ARMV8_NEON)
```

Comment:
What happens if the builder has Neon and and the Debian machine wil not have it? Is it a valid configuration for Debian? Can't Neon extension be softly turned off on the processor?

Reply:
Actually very good question. I assume that if it's build with NEON code and run in machine that does not have it it will crash hardly (or GCC can handle it). As far as I know I don't you can't turn NEON on from CPU it's kind you turn off math unit in x86. If you compile your own kernel you can opt-it out.
If you are talking about Debian `armel` then it should work if machine has or not as it's emulated. But if you have ARMv8-a and run it in ARMV8-r without NEON then as said you are probably out of luck.